sdk/resourcemanager/Azure.ResourceManager/src/Common/Custom/Models/TrackedResourceData.cs

```diff
@@ -36,6 +36,13 @@ protected TrackedResourceData(ResourceIdentifier id, string name, ResourceType r
             Location = location;
         }
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="TrackedResourceData"/> for deserialization.
+        /// </summary>
+        protected TrackedResourceData()
```

Comment:
The constructor on L33 is marked as the serialization constructor.  It may be worth a bit more detail in the comment here on why this is needed, lest a future someone decide it can be removed.

Reply:
These models will all be generated once the generator pr https://github.com/Azure/autorest.csharp/pull/3434 gets merged in.  I will make this comment change there.